### PR TITLE
Enable NFS support for Photon OS template based CSE clusters

### PR DESCRIPTION
**Pull request by sompa**
VCDA-614 :  Enable NFS support for Photon OS

This PR addressed issue #113. Added related customization to photon-OS scripts to enable NFS PV for CSE clusters.
Manually tested the following : Creation of K8 clusters using photon template with NFS enabled, checked that the nfs server deployed in the K8 cluster had the nfs-utils package installed ('tdnf info recent' or tdnf info nfs-utils ), verified that the NFS share was added to the /etc/export on the NFS server, deployed a busy box application using PVC and validated the deployment. 

@sahithi @rocknes @harshneelmore @sakthisunda @andrew-ni 
Signed-off-by: Sompa Malakar <malakars@vmware.com>

---
This change is [Reviewable](https://reviewable.io/reviews/vmware/container-service-extension/154)

### CODE REVIEW

File: scripts/nfsd-photon-v2.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -e
```
Comment (sahithi) [inline scripts/nfsd-photon-v2.sh]: Change the documentation to reflect NFS PV support is available on Photon as well. It currently says not supported.

File: scripts/cust-photon-v2.sh [hunk 2]
```diff
@@ -44,6 +44,13 @@ docker pull weaveworks/weave-kube:2.3.0
 export kubever=$(kubectl version --client | base64 | tr -d '\n')
 wget --no-verbose -O weave.yml "https://cloud.weave.works/k8s/net?k8s-version=$kubever&v=2.3.0"
 
+
+#Installing NFS
+echo 'installing required software for NFS'
+tdnf -y install nfs-utils
+systemctl stop nfs-server.service
+systemctl disable nfs-server.service
```
Comment (sahithi) [inline scripts/cust-photon-v2.sh]: Have you tested the case if pods running on ubuntu are able to access volumes on Photon? This is a valid use case now.

**Comment by sompa** [comment]
___
*[scripts/nfsd-photon-v2.sh, line 2 at r1](https://reviewable.io/reviews/vmware/container-service-extension/154#-LRAaJLTivtwsA_xlCM4-r1-2:-LRDsHEK0ACDeimJQ5iV:b-80hpij) ([raw file](https://github.com/vmware/container-service-extension/blob/d1c10fe7a24989d83c8321c5b86c626971e0397b/scripts/nfsd-photon-v2.sh#L2)):*
<details><summary><i>Previously, sahithi (Sahithi Ayloo) wrote…</i></summary><blockquote>

Change the documentation to reflect NFS PV support is available on Photon as well. It currently says not supported.
</blockquote></details>

Its not mentioned in the documentation (github.io) that we only support Ubuntu. If you happen to see any place that needs change in the doc for this, could you please point me to it ?

**Comment by sompa** [comment]
___
*[scripts/nfsd-photon-v2.sh, line 7 at r1](https://reviewable.io/reviews/vmware/container-service-extension/154#-LRAbNR4EQS8QvUBfSCP:-LRDsaQ9919WCZAbom-I:b-896fix) ([raw file](https://github.com/vmware/container-service-extension/blob/d1c10fe7a24989d83c8321c5b86c626971e0397b/scripts/nfsd-photon-v2.sh#L7)):*
<details><summary><i>Previously, rocknes wrote…</i></summary><blockquote>

Missing EOL on the last line.
</blockquote></details>

Done.

**Comment by sompa** [comment]
___
*[scripts/cust-photon-v2.sh, line 52 at r1](https://reviewable.io/reviews/vmware/container-service-extension/154#-LRAaJLTivtwsA_xlCM3-r1-52:-LREIgU--48QakXFrODC:by60s4f) ([raw file](https://github.com/vmware/container-service-extension/blob/d1c10fe7a24989d83c8321c5b86c626971e0397b/scripts/cust-photon-v2.sh#L52)):*
<details><summary><i>Previously, sahithi (Sahithi Ayloo) wrote…</i></summary><blockquote>

Have you tested the case if pods running on ubuntu are able to access volumes on Photon? This is a valid use case now.
</blockquote></details>

Have tested out  pods running on ubuntu are able to access volumes on Photon and vice versa as well.

**Comment by sompa** [comment]
___
*[scripts/cust-photon-v2.sh, line 52 at r1](https://reviewable.io/reviews/vmware/container-service-extension/154#-LRAaJLTivtwsA_xlCM3-r1-52:-LREVG3h4QQ_IM7KNp1P:b-896fix) ([raw file](https://github.com/vmware/container-service-extension/blob/d1c10fe7a24989d83c8321c5b86c626971e0397b/scripts/cust-photon-v2.sh#L52)):*
<details><summary><i>Previously, sompa (malakars) wrote…</i></summary><blockquote>

Have tested out  pods running on ubuntu are able to access volumes on Photon and vice versa as well. 
</blockquote></details>

Done.

**Pull request merged**